source/includes/_examples.md

```diff
@@ -0,0 +1,219 @@
+# Examples
+
+## CDC Guideline for Prescribing Opioids for Chronic Pain - Request
```

Comment:
Nit: Maybe we should make the h2 'CDC Guideline for Prescribing Opioids for Chronic Pain' and then both Request and Response can be h3 elements?